examples/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(frozen.examples.srcs enum_to_string enum_to_string_hash pixel_art static_assert)
+
+if ("${CMAKE_CXX_COMPILER_ID}" MATCHES ".*Clang")
+    list(APPEND frozen.examples.srcs html_entities_map)
+endif()
+
+foreach(src ${frozen.examples.srcs})
+    add_executable(frozen.tests.${src} ${src}.cpp)
+    target_link_libraries(frozen.tests.${src} PUBLIC frozen::frozen)
+endforeach()
+
+if ("${CMAKE_CXX_COMPILER_ID}" MATCHES ".*Clang")
```

Comment:
I initially thought the if-statement was redundant, but I see that this target exists only for clang targets. 

I might be a bit more explicit and say
```cmake
if(TARGET frozen.tests.html_entities_map)
  ...
endif()
```

But otherwise, looks solid 👍